### PR TITLE
Pie Baker

### PR DESCRIPTION
**Pull request by bttq-dll**
The Pie Baker Plugin automates baking pies in OSRS using the Lunar spellbook's Bake Pie spell. 
It allows users to select from various raw pies, automatically withdraws them from the bank, casts the spell to bake them, and deposits the cooked pies. 
The plugin supports tick-perfect casting for efficiency and tracks Magic and Cooking XP per hour. 
Users can view the selected pie and XP rates via an in-game overlay. 

Instructions:
Ensure you have the required runes in your inventory and start the bot at any bank, preferably a populated one like the Grand Exchange.

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/microbot/piebaker/PieBakerPlugin.java [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+import net.runelite.client.plugins.microbot.util.magic.Rs2Magic;
+import net.runelite.client.ui.overlay.OverlayManager;
+
+import javax.inject.Inject;
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.Map;
+
+@PluginDescriptor(
+        name = "Pie Baker",
```
Comment (chsami) [inline runelite-client/src/main/java/net/runelite/client/plugins/microbot/piebaker/PieBakerPlugin.java]: Add your name in the plugindescriptor and prepend it in the script name, this way people know who made the plugin and can contact you if necessary.

**Review by chsami — CHANGES_REQUESTED** [review]

**Review by bttq-dll — COMMENTED** [review]
Updated

**Review by bttq-dll — COMMENTED** [review]
i see B is already taken so i choosed J

**Comment by bttq-dll** [comment]
Added **Flippers Chaser**
This plugin automates the process of obtaining Flippers from Mogres by utilizing fishing explosives and handling combat scenarios. Additionally, it integrates Discord notifications to keep players informed about the plugin's success or failure.

- Automated Combat
- Prayer Management
- Food Consumption
- Discord Notifications
- Automatic Logout or Teleport

The plugin will automatically use fishing explosives on the "Ominous Fishing Spot" to spawn Mogres.
Upon spawning, the plugin will engage and kill the Mogres.
The plugin will loot Flippers and then teleport to your house or log out if a house teleport tab is unavailable.
Discord notifications will inform you of the outcome, including a screenshot for success or failure.

Added also discord notification embeds:
`[Success] **Player name** has obtained **Flippers** successfully!
`
`[Failure] **Player name** has no food in inventory, logging out.
`
Contributions are welcome!

**Comment by bttq-dll** [comment]
I joined discord 
image

**Review by chsami — APPROVED** [review]

**Comment by chsami** [comment]
There are still some errors in the flipper plugin about discord.

**Comment by bttq-dll** [comment]
Nested DiscordWebhook class for simplicity in Plugin file

**Pull request merged**